### DISCUSSION TITLE
i am getting deployment errors (react-script command not found error

### DISCUSSION THREAD

**Discussion by Blueavian9** [vercel/community#1701] [Help] [answered]
### Summary

i have 2 building errors when trying to deploy my website to my github with vercel 
 1) sh: react-scripts: command not found.
 2) Error: Command "react-scripts build" excited with 127
ps. i would kindly appreciate the feedback in advanced 

### Additional information

here is an image: 

![image (2) (1)](https://user-images.githubusercontent.com/68366917/223025354-b86c05a8-597b-48e0-b4fc-1d7ecdc8e64b.png)

### Example

https://vercel.com/blueavian9/demo-app-rcyh/DEHJnJ5PtCFtk3KUsfdfkvcGR8E2

**Comment by shannonZHONG** [comment]
Hello, do you know how to solve the issue? Thanks

**Comment by amyegan** [comment, marked as answer]
Please make sure all required packages are included in `package.json`. You may need to re-generate the `package-lock.json` file.

https://stackoverflow.com/questions/47612580/react-scripts-command-not-found

**Comment by Jemish1306** [comment]
any one  know how to solve this?
